.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2.1
+
+orbs:
+  # I'm using a fork of the circleci/ruby orb since the upstream version's
+  # `install-deps` command assumes we have a Gemfile.lock to use in the cache
+  # key which we don't. There is an open PR to allow completely custom cache
+  # keys which hopefully will be merged soon:
+  # https://github.com/CircleCI-Public/ruby-orb/pull/46
+  ruby: cstyles/ruby@dev:install-deps-custom-key
```

Comment:
This is a "dev" tag/version so it will expire in 90 days if it isn't updated. I'm hoping the PR mentioned in the comment above (or something similar) will be merged before then. I'll set a reminder to check in on this later.